docs/advanced-topics/advanced-features.rst

```diff
@@ -461,7 +472,7 @@ example:
 
     token: <my token>
     server-url: https://dockstore.org/api
-    cromwell-version: 50
+    cromwell-version: 39
```

Comment:
I think I commented on the last PR, but I would not have `cromwell-version`at all. Most people probably won't have it, since it's not in the default config file.

Reply:
Removed cromwell-version